.github/workflows/CI.yaml

```diff
@@ -92,7 +92,7 @@ jobs:
         run: ./gradlew ktlintCheck detekt --no-daemon --stacktrace
         if: always()
       - name: Execute Backend unit tests
```

Comment:
Können wir den step vllt umbenennen?
Hier laufen ja mehr als nur die unit tests des backends.

Reply:
Spring Service unit tests?

Reply:
Execute unit test of microservices?
Excecute unit test of backend services?

Aber eine Anmerkung dazu. Sind nicht alle Microservices, die nicht das Frontend betreffen, irgendwie Backend Services?

Reply:
"Execute unit tests of all spring services" fänd ich gut,   weil frontend "unit" tests ja nicht ausgeführt werden

Und @stewal19 ja es sind backend services, aber ich finde wenn da wie jetzt so deutlich "Backend" steht interpretiert man das bei dem wording in unserem Team als den backend microservices selbst, und nicht die anderen Microservices die wir mittlerweile haben. War zumindest bei mir so als ich dieses file nochmals durchgelesen habe beim review jetzt.
Was meinst du?

Reply:
Für mich ist die Änderung in Ordnung. Habe da keine starke Meinung zu

Reply:
Done

Reply:
👍